### PR TITLE
Port to Python3

### PR DESCRIPTION
**Pull request by pratyush3124**
Port to Python3

### CODE REVIEW

File: scorescreen.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ def make(self, gameDisplay, back, score, fruitscore):
         back.set_alpha(225)
 
         font_path = "fonts/Arimo.ttf"
-        font_size = 55
+        font_size = 50
```
Comment (Hrishi1999) [inline scorescreen.py]: Why is this changed?

File: scorescreen.py [hunk 2]
```diff
@@ -158,7 +158,7 @@ def make(self, gameDisplay, back, score, fruitscore):
             gameDisplay.blit(plate, (430, 40))
 
             head1 = font1.render(_("GAME OVER!"), 1, (white))
-            gameDisplay.blit(head1, (440, 40))
+            gameDisplay.blit(head1, (440, 50))
```
Comment (Hrishi1999) [inline scorescreen.py]: Here too.

**Comment by quozl** [comment]
Thanks.  Reviewed.  

Does not comply with our coding convention; new flake8 warnings appear;

```log
./rules.py:46:18: E211 whitespace before '('
./welcomescreen.py:51:18: E211 whitespace before '('
./scorescreen.py:49:18: E211 whitespace before '('
./main.py:54:18: E211 whitespace before '('
```

(But please don't fix other flake8 warnings in this pull request unless they are reported against lines you changed.)

Please remove the merge commit from the branch, if you get a chance.  Not significant, just unnecessary.

And you'll find it easier in future not to use your _master_ branch.  See our [guide to contributing](https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md).

**Comment by pratyush3124** [comment]
Yes thank you

File: scorescreen.py (hunk 1 above)
Comment (pratyush3124) [reply scorescreen.py]: Sir I dint change it. It was done before

**Comment by quozl** [comment]
@Hrishi1999, the commit is marked as authored by @nswarup14, so best ask them.

**Comment by quozl** [comment]
See https://github.com/sugarlabs/stick-hero-activity/pull/29
